lib/src/tabs/Tabs.tsx

```diff
@@ -19,102 +49,232 @@ const DxcTabs = ({
   iconPosition = "top",
   tabIndex = 0,
 }: TabsPropsType): JSX.Element => {
-  const [innerActiveTabIndex, setInnerActiveTabIndex] = useState(defaultActiveTabIndex ?? 0);
   const colorsTheme = useTheme();
+  const [innerActiveTabIndex, setInnerActiveTabIndex] = useState(defaultActiveTabIndex ?? 0);
   const hasLabelAndIcon = tabs && tabs.filter((tab) => tab.label && tab.icon).length > 0;
+  const [activeIndicatorWidth, setActiveIndicatorWidth] = useState(0);
+  const [activeIndicatorLeft, setActiveIndicatorLeft] = useState(0);
+  const [translateScroll, setTranslateScroll] = useState(0);
+  const [scrollRightEnabled, setScrollRightEnabled] = useState(true);
+  const [scrollLeftEnabled, setScrollLeftEnabled] = useState(false);
+  const [countClick, setCountClick] = useState(0);
+  const [totalTabsWidth, setTotalTabsWidth] = useState(0);
+  const [currentFocusIndex, setCurrentFocusIndex] = useState(0);
+  const [temporalFocusIndex, setTemporalFocusIndex] = useState(0);
+  const [minHeightTabs, setMinHeightTabs] = useState(0);
+  const refTabs = useRef([]);
+  const refTabList = useRef(null);
+  const viewWidth = useResize(refTabList);
+  const translatedLabels = useTranslatedLabels();
+
+  const enabledIndicator = useMemo(() => viewWidth < totalTabsWidth, [viewWidth]);
 
-  const handleChange = (event, newValue) => {
+  useEffect(() => {
+    let sumWidth = refTabs?.current?.reduce(function (count, obj) {
+      return count + obj.offsetWidth;
+    }, 0);
+    setTotalTabsWidth(sumWidth);
+    setActiveIndicatorWidth(refTabs?.current[activeTabIndex ?? innerActiveTabIndex]?.offsetWidth);
+    setActiveIndicatorLeft(refTabs?.current[activeTabIndex ?? innerActiveTabIndex]?.offsetLeft);
+  }, [refTabs]);
+
+  useEffect(() => {
+    setMinHeightTabs(refTabList?.current?.offsetHeight + 1);
+  }, [refTabList]);
+
```

Comment:
I think we can put `setMinHeightTabs(refTabList?.current?.offsetHeight + 1);` inside the useEffect above.

Reply:
Because refTabList and refTabs are different, they should be in different useEffect to prevent the executing of the rest of the code that should not be executed.